### ISSUE TITLE
grpc fails to build on centos buildbot

### ISSUE THREAD

**Issue by crowell** [apache/incubator-pagespeed-mod#1334]
```
  cc '-DCHROMIUM_REVISION=256281' '-D_GLIBCXX_USE_CXX11_ABI=0' '-DV8_DEPRECATION_WARNINGS' '-DBLINK_SCALE_FILTERS_AT_RECORD_TIME' '-D_FILE_OFFSET_BITS=64' '-DPAGESPEED_SUPPORT_POSIX_SHARED_MEM' '-DGTEST_HAS_RTTI=1' '-DCHROMIUM_BUILD' '-DTOOLKIT_VIEWS=1' '-DUI_COMPOSITOR_IMAGE_TRANSPORT' '-DUSE_AURA=1' '-DUSE_CAIRO=1' '-DUSE_GLIB=1' '-DUSE_DEFAULT_RENDER_THEME=1' '-DUSE_LIBJPEG_TURBO=1' '-DUSE_NSS=1' '-DUSE_X11=1' '-DUSE_CLIPBOARD_AURAX11=1' '-DENABLE_ONE_CLICK_SIGNIN' '-DUSE_XI2_MT=2' '-DENABLE_REMOTING=1' '-DENABLE_WEBRTC=1' '-DENABLE_PEPPER_CDMS' '-DENABLE_CONFIGURATION_POLICY' '-DENABLE_INPUT_SPEECH' '-DENABLE_NOTIFICATIONS' '-DENABLE_NEW_GAMEPAD_API=1' '-DUSE_UDEV' '-DENABLE_EGLIMAGE=1' '-DENABLE_TASK_MANAGER=1' '-DENABLE_EXTENSIONS=1' '-DENABLE_PLUGIN_INSTALLATION=1' '-DENABLE_PLUGINS=1' '-DENABLE_SESSION_SERVICE=1' '-DENABLE_THEMES=1' '-DENABLE_AUTOFILL_DIALOG=1' '-DENABLE_BACKGROUND=1' '-DENABLE_AUTOMATION=1' '-DENABLE_GOOGLE_NOW=1' '-DCLD_VERSION=2' '-DENABLE_FULL_PRINTING=1' '-DENABLE_PRINTING=1' '-DENABLE_SPELLCHECK=1' '-DENABLE_CAPTIVE_PORTAL_DETECTION=1' '-DENABLE_APP_LIST=1' '-DENABLE_SETTINGS_APP=1' '-DENABLE_MANAGED_USERS=1' '-DENABLE_MDNS=1' '-DOPENSSL_SMALL' '-DNDEBUG' '-DNVALGRIND' '-DDYNAMIC_ANNOTATIONS_ENABLED=0' -Ithird_party/grpc/src -Ithird_party/grpc/src/include -Ithird_party/chromium/src -Ithird_party/boringssl/src/include  -fstack-protector --param=ssp-buffer-size=4 -pthread -fno-exceptions -fno-strict-aliasing -Wno-unused-parameter -Wno-missing-field-initializers -fvisibility=hidden -pipe -fPIC -Wno-unused-local-typedefs -fexceptions -fasynchronous-unwind-tables -D_FORTIFY_SOURCE=2 -std=c99 -Wno-format -Wno-unused-result -O2 -fno-ident -fdata-sections -ffunction-sections -funwind-tables  -MMD -MF out/Release/.deps/out/Release/obj.target/grpc_core/third_party/grpc/src/src/core/iomgr/pollset_multipoller_with_epoll.o.d.raw  -c -o out/Release/obj.target/grpc_core/third_party/grpc/src/src/core/iomgr/pollset_multipoller_with_epoll.o third_party/grpc/src/src/core/iomgr/pollset_multipoller_with_epoll.c
third_party/grpc/src/src/core/iomgr/pollset_multipoller_with_epoll.c: In function ‘epoll_become_multipoller’:
third_party/grpc/src/src/core/iomgr/pollset_multipoller_with_epoll.c:294:3: warning: implicit declaration of function ‘epoll_create1’ [-Wimplicit-function-declaration]
   h->epoll_fd = epoll_create1(EPOLL_CLOEXEC);
   ^
third_party/grpc/src/src/core/iomgr/pollset_multipoller_with_epoll.c:294:31: error: ‘EPOLL_CLOEXEC’ undeclared (first use in this function)
   h->epoll_fd = epoll_create1(EPOLL_CLOEXEC);
                               ^
third_party/grpc/src/src/core/iomgr/pollset_multipoller_with_epoll.c:294:31: note: each undeclared identifier is reported only once for each function it appears in
make: *** [out/Release/obj.target/grpc_core/third_party/grpc/src/src/core/iomgr/pollset_multipoller_with_epoll.o] Error 1
```

**Comment by crowell** [comment]
tracked here https://github.com/grpc/grpc/issues/7147

**Comment by jeffkaufman** [comment]
Is this still broken?  Is `-DGPR_MANYLINUX1` all we need(ed)?

**Comment by hillsp** [comment]
I believe this should have been fixed with https://github.com/pagespeed/mod_pagespeed/commit/c7d0c2ea9ec158e7dbc0b9fae7f578c1afe8ed78

**Comment by crowell** [comment]
yes, this was fixed in c7d0c2e, upgrading to 0.15 and `-DGPR_MANYLINUX1` were needed. 0.13 had no support for the older kernel.
